tahoe_auth0/api_client.py

```diff
@@ -73,6 +78,29 @@ def get_connection(self):
 
         return "con-{}".format(org_id)
 
+    def change_password_via_reset_for_db_connection(self, email):
+        """
+        Start password reset for a specific email via /dbconnections/change_password .
+        """
+        connection = self.get_connection()
+
+        logger.debug("Starting password reset for user %s in connection: %s", email, connection)
+
+        client_id, _client_secret = helpers.get_client_info()
+        resp = requests.post(
```

Comment:
Can you guess what I'm going to suggest on this? ;)

Reply:
yup!!